plugin/trino-bigquery/pom.xml

```diff
@@ -19,6 +19,14 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>google-cloud-bigquerystorage</artifactId>
```

Comment:
Can we move this to `dependencies` instead of `dependencyManagement`. That way we don't have to specify the version (since we immport the libraries-bom already).

Reply:
The storage API is already present in the `dependencies` and, as far as I can tell, not pulled in by the libraries-bom -- the read and write APIs are a separate service from the main BigQuery service, and the clients are packaged separately. I'm not an expert on Maven dependency resolution (😄), but if I don't explicitly specify a version here then I get an older version of the bigquerystorage client than I'm looking for:

```
kmj-macbookpro:trino kmj$ ./mvnw --projects plugin/trino-bigquery dependency:list
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------------< io.trino:trino-bigquery >-----------------------
[INFO] Building trino-bigquery 365-SNAPSHOT
[INFO] ----------------------------[ trino-plugin ]----------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:3.1.2:list (default-cli) @ trino-bigquery ---
[INFO] The following files have been resolved:
```
```
[INFO]    com.google.cloud:google-cloud-bigquerystorage:jar:2.1.0:compile -- module google.cloud.bigquerystorage (auto)
```

Or am I misunderstanding your suggestion?

Reply:
You're correct. libraries-bom latest version is 24.0.0 which still depends on bigquerystorage 2.4.2.

Add a comment like `TODO remove when libraries-bom includes bigquerystorage >= 2.5.1`.

BTW any reason to use 2.5.1 instead of 2.6.3 (latest)?

Reply:
Actually, scratch that. I'm looking to include [this PR](https://github.com/googleapis/java-bigquerystorage/pull/1257) here, but it looks like the associated commit is already present in 2.1.0 -- I guess that no POM changes are required (since I assume you'd prefer to avoid bringing the bigquerystorage bits out of sync with `libraries-bom` for no reason). I'll revert this file.

Reply:
Done.